python/composio/client/collections.py

```diff
@@ -1316,6 +1336,39 @@ def _serialize_auth(auth: t.Optional[CustomAuthObject]) -> t.Optional[t.Dict]:
         ]
         return data
 
+    def search_for_a_task(
```

Comment:
The `search_for_a_task` method lacks input validation for `min_actions_per_task` and `max_actions_per_task`. Consider adding validation to ensure min is not greater than max when both are provided.